html/pfappserver/root/node/tab_WMI_process.tt

```diff
@@ -1,5 +1,5 @@
 [% IF rules.id == "Process_Running" %]
-  [% IF rules.item_exist == "No" %]
+  [% IF rules.item_exist == "No" || rules.item_exist == "Access denied" %]
     <span> Error while getting running process. [% rules.item_exist | html %]</span><br/>
```

Comment:
Can you localize string ? [% l(' Error while getting running process. ') %]
Item scanned, Is Present, Name, Version, too. (in this file)